coriolis/constants.py

```diff
@@ -45,8 +45,13 @@
 TASK_TYPE_DELETE_REPLICA_DISK_SNAPSHOTS = "DELETE_REPLICA_DISK_SNAPSHOTS"
 TASK_TYPE_RESTORE_REPLICA_DISK_SNAPSHOTS = "RESTORE_REPLICA_DISK_SNAPSHOTS"
 TASK_TYPE_GET_OPTIMAL_FLAVOR = "GET_OPTIMAL_FLAVOR"
-TASK_TYPE_VALIDATE_MIGRATION_INPUTS = "VALIDATE_MIGRATION_INPUTS"
-TASK_TYPE_VALIDATE_REPLICA_INPUTS = "VALIDATE_REPLICA_INPUTS"
+TASK_TYPE_VALIDATE_MIGRATION_INPUTS_SOURCE = (
+    "VALIDATE_MIGRATION_INPUTS_SOURCE")
+TASK_TYPE_VALIDATE_MIGRATION_INPUTS_DESTINATION = (
+    "VALIDATE_MIGRATION_INPUTS_DESTINATION")
+TASK_TYPE_VALIDATE_REPLICA_INPUTS_SOURCE = "VALIDATE_REPLICA_INPUTS_SOURCE"
+TASK_TYPE_VALIDATE_REPLICA_INPUTS_DESTINATION = (
+    "VALIDATE_REPLICA_INPUTS_DESTINATION")
 TASK_TYPE_VALIDATE_REPLICA_DEPLOYMENT_INPUTS = (
     "VALIDATE_REPLICA_DEPLOYMENT_INPUTS")
 TASK_TYPE_UPDATE_REPLICA = "UPDATE_REPLICA"
```

Comment:
@ionutbalutoiu While you're at it, could you please also split up `TASK_TYPE_UPDATE_REPLICA ` as well? (it also does calls to both providers in a single task)

Reply:
Yep. I'll do it. I'll add this as a separate commit to this PR.